packages/app-project/src/screens/ProjectAboutPage/components/AboutDropdownNav.js

```diff
@@ -14,7 +14,7 @@ export const AboutDropContent = ({ aboutNavLinks, router }) => {
   const baseUrl = `/projects/${owner}/${project}/about`
   
   return (
-    <Nav gap="xsmall" background="white">
+    <Nav gap='xsmall' background={{ dark: 'dark-3', light: 'neutral-6' }}>
```

Comment:
The `dark-3` color background here is a bit hard to distinguish from the rest of the page. In the past, Becky has had us use `dark-5` in these cases and you can see that in action with the shared components' `Modal`. 

Here's how this looks:

<img width="645" alt="Screen Shot 2021-07-19 at 11 00 37 AM" src="https://user-images.githubusercontent.com/5016731/126190811-4a6eb8f7-4962-4c7d-9742-9fa90519a554.png">

```suggestion
    <Nav gap='xsmall' background={{ dark: 'dark-5', light: 'neutral-6' }}>
```